go.mod

```diff
@@ -7,11 +7,11 @@ require (
 	github.com/axw/gocov v1.1.0
 	github.com/edaniels/golinters v0.0.5-0.20220906153528-641155550742
 	github.com/fullstorydev/grpcurl v1.8.9
-	github.com/golangci/golangci-lint v1.51.2
+	github.com/golangci/golangci-lint v1.54.0
 	github.com/rhysd/actionlint v1.6.24
 	go.uber.org/multierr v1.11.0
-	go.viam.com/api v0.1.281
-	go.viam.com/rdk v0.25.0
+	go.viam.com/api v0.1.289
+	go.viam.com/rdk v0.26.0-rc0.0.20240426210745-a81e17bcc2ec
```

Comment:
fine with pinning v0.26.0(or w/e it ends up being before I merge). back with cartographer/cloudslam I would have to pin to a non-tagged commit of rdk in order to ensure they released together with cartographer, but since thats not an issue for this module I am fine with waiting. 

I'll probably also do a new release of this module after this pr gets merged as well